docs/UserGuide.md

```diff
@@ -20,20 +30,51 @@ Vax@NUS is a **one stop management app to efficiently track and schedule COVID-1
 
 1. Copy the file to the folder you want to use as the _home folder_ for your Vax@NUS application.
 
-1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
+1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. 
+   Sample data will be displayed to show you how the user interface of the app looks.<br>
    ![Ui](images/Ui.png)
 
 1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
-   Some example commands you can try:
 
-   * **`list`** : Lists all data.
-   * **`exit`** : Exits the app.
+1. Before you begin using Vax@NUS, here are some example commands you can use to work with the sample data to familarize yourself:
+   * **`add A1234567X n/John Doe f/COM p/98765432 e/johnd@example.com a/John street, block 123, #01-01 s/vaccinated m/peanut allergy r/RVRC
+     `** <br>
+     Adds a student named John Doe, identified by matriculation number A1234567X, to the student records, along with relevant personal details.
+   * **`addAppt A1234567X d/2021-12-13 ts/13:00
+     `** <br>
+     Creates a new vaccination appointment for John Doe, identified with matriculation number A1234567X, at the specified date and time. 
+   * **`statsAppt`** <br>
+    Tells you the number of appointments in the past week, and in the next week.
+   * **`delete A1234567X`** <br>
+    Deletes the information for John Doe, identified by his matriculation number, from the student records. Note that since
+     John Doe also has a vaccination appointment created, the appointment will also be deleted. 
 
 1. Refer to the [Features](#features) below for details of each command.
 
+1. When you are ready, you can remove all sample data with the `clear` command and begin adding real data. 
+
 --------------------------------------------------------------------------------------------------------------------
 <br> 
 
+## Prefix Table
+
+This table shows you which prefix should be used for the different pieces of information.
+
+Prefix | Keyword           | As Used
+-------|-------------------|----------                  
+**n/** | NAME              | `n/NAME`
+**f/** | FACULTY           | `f/FACULTY`
+**p/** | PHONE_NUMBER      | `p/PHONE_NUMBER`
+**e/** | EMAIL             | `e/EMAIL`
+**a/** | ADDRESS           | `a/ADDRESS`
+**s/** | VACCINATION_STATUS| `s/VACCINATION_STATUS`
+**m/** | MEDICAL_DETAILS   | `m/MEDICAL_DETAILS`
+**r/** | SCHOOL_RESIDENCE  | `r/SCHOOL_RESIDENCE`
+
+Please also refer to the [Input Formats](#input-formats) below to find out how the information
```

Comment:
Would "...refer to the Input Formats section below..." or "... refer to Input Formats below..." be better?

Reply:
Added